Sources/KituraWebSocket/WSFrameParser.swift

```diff
@@ -112,8 +112,7 @@ struct WSFrameParser {
         switch lengthByte {
         case 126:
             if length - from >= 3 {
-                let networkOrderedUInt16 = UnsafeRawPointer(bytes+from+1).assumingMemoryBound(to: UInt16.self)[0]
-                
+                 let networkOrderedUInt16 = bytes.load(fromByteOffset: from+1, as: UInt16.self)
```

Comment:
It looks like the tests intermittently fail with a misaligned load here.  Maybe when tested locally, the loads always happened (by chance) to be aligned?